sections/exchange/TradeCard/CurrencyCard/CurrencyCard.tsx

```diff
@@ -120,17 +120,16 @@ const CurrencyCard: FC<CurrencyCardProps> = ({
 								<FlexDivRowCentered>
 									<CurrencyAmountValue data-testid="amount-value">
 										{tradeAmount != null
-											? formatCurrency(selectedPriceCurrency.name, tradeAmount, {
+											? formatCurrency(selectedPriceCurrency.name as CurrencyKey, tradeAmount, {
 													sign: selectedPriceCurrency.sign,
 											  })
 											: null}
 									</CurrencyAmountValue>
 									<Slippage>
 										{!isLoading &&
 											slippagePercent != null &&
-											!slippagePercent.isNaN() &&
-											slippagePercent.isFinite() &&
-											slippagePercent.isNegative() &&
+											slippagePercent.gt(10000) &&
```

Comment:
did the logic before not make sense here?

Reply:
if `slippagePercent > 10000 AND slippagePercent < 0` isn't this always false or am I reading this wrong?

Reply:
the reason it was changed is because `wei` does not have a concept of `NaN` or `Infinity`, literally just integers. If you do anything that would result in a NaN or infinity, you get an exception.

upon examination, `slippagePercent` is actually the ratio (100% = 1) of estimated price to the actual price. This value is negative if slippage exists, and based on existing code this should only be shown if there is unfavorable slippage. Therefore, the condition we want to show based on what exists before is `slippagePercent.lt(0)` and nothing else.

Reply:
making change for this.

Reply:
Yeah I'm confused